generator-dialtone/generators/vue3/templates/story.ejs

```diff
@@ -5,8 +5,8 @@
     as the storybook control defined in the corresponding `.story.js` file.
   -->
   <<%= componentNameKebab %>
-    :generic-prop="$attrs.genericProp"
-    @generic-event="$attrs.onGenericEvent"
+    :generic-prop="genericProp"
+    @generic-event="onGenericEvent"
```

Comment:
We need to still include attrs don't we? or did you make it so this is handled elsewhere?

Reply:
Right, forgot to update templates. 

Fixed on latest commit.